config/routes.rb

```diff
@@ -167,23 +171,34 @@
       resources :user, controller: :users, except: EXCEPT
     end
 
-    # Ahoy email tracking
+    ########################################
+    # Rails engines
+
+    # AhoyEmail provides email tracking features
     mount AhoyEmail::Engine, at: '/ahoy'
 
-    # Blazer (web stats dashboard)
+    # Blazer provides charts and dashboards in the admin area
     mount Blazer::Engine, at: '/admin/stats'
 
-    # CKEditor (WYSIWYG editor used on various admin pages)
+    # CKEditor provides the WYSIWYG editor used in the admin area
     mount Ckeditor::Engine, at: '/admin/ckeditor'
 
-    # Mailer preview features
+    # RailsEmailPreview provides previews of site emails in the admin area
     mount RailsEmailPreview::Engine, at: '/admin/email-previews'
 
-    # Letter Opener webmail UI for dev environment
+    # LetterOpener provides a webmail UI for viewing sent emails (in dev only)
     mount LetterOpenerWeb::Engine, at: 'letter-opener' if Rails.env.development?
 
-    # This catch-all route passes through to the Pages controller, allowing
-    # sites to have top-level pages (e.g. /foo instead of /pages/foo).
+    ########################################
+    # ShinyCMS plugins
+
+    # ShinyForms: generic form handlers
+    mount ShinyForms::Engine, at: '/'
```

Comment:
Consider wrapping this with a conditional to check if the plugin is enabled, e.g.

```suggestion
    mount ShinyForms::Engine, at: '/' if defined?(ShinyForms)
```